docs/DeveloperGuide.md

```diff
@@ -576,6 +656,25 @@ testers are expected to do more _exploratory_ testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Finding a contact
+
+1. Finding contacts by names, tags, or companies
+
+   1. Prerequisites: List all persons using the `list` command. There should be multiple persons in the address book with varying names, tags, and company affiliations.
```

Comment:
Maybe instead of requiring it, it's better to say **ideally** there should be multiple contacts...
Because it still works without having multiple contacts